events/tests/snapshots/snap_test_api.py

```diff
@@ -69,25 +69,14 @@
 }
 
 snapshots["test_add_event_staff_user 1"] = {
-    "data": {
-        "addEvent": {
-            "event": {
-                "capacityPerOccurrence": 30,
-                "duration": 1000,
-                "image": "",
-                "participantsPerInvite": "FAMILY",
-                "publishedAt": None,
-                "translations": [
-                    {
-                        "description": "desc",
-                        "languageCode": "FI",
-                        "name": "Event test",
-                        "shortDescription": "Short desc",
-                    }
-                ],
-            }
+    "data": {"addEvent": {"event": None}},
+    "errors": [
+        {
+            "locations": [{"column": 7, "line": 12}],
+            "message": "Cannot return null for non-nullable field EventNode.image.",
```

Comment:
Hmm, this doesn't seem right :)

Reply:
Fixed, the field is not null in DB so `resolve_image` should return empty string instead of `None`